polygeist/tools/cgeist/Test/Verification/sycl/functions.cpp

```diff
@@ -16,7 +16,7 @@
 // CHECK: !sycl_id_2_ = !sycl.id<2>
 // CHECK: !sycl_item_2_1_ = !sycl.item<[2, true], (!sycl.item_base<[2, true], (!sycl.range<2>, !sycl.id<2>, !sycl.id<2>)>)>
 
-// CHECK: func.func @_Z8method_1N2cl4sycl4itemILi2ELb1EEE(%arg0: !sycl_item_2_1_) attributes {llvm.linkage = #llvm.linkage<external>} {
```

Comment:
For our reference, these changes are due to https://github.com/intel/llvm/commit/3d2b25e13ecd2d941b436fe903a5d456037a1bd9.